src/main.zig

```diff
@@ -544,7 +544,7 @@ const usage_build_generic =
     \\  --verbose-llvm-ir            Enable compiler debug output for LLVM IR
     \\  --verbose-cimport            Enable compiler debug output for C imports
     \\  --verbose-llvm-cpu-features  Enable compiler debug output for LLVM CPU features
-    \\  --debug-log [scope]          Enable printing debug/info log messages for scope
+    \\  --debug-log [scopes]         Enable printing debug/info log messages for one or more scopes (CSV)
```

Comment:
The intended usage of multiple scopes is like this:

```
--debug-log scope1 --debug-log scope2
```

nice and simple. I don't see any reason to add additional syntax to a debug option.